users/views.py

```diff
@@ -13,3 +18,31 @@ def register(request):
     return Response(
         serializer.validated_data,
     )
+
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def get_events(request):
+   
+   events = Event.objects.filter(creator=request.user)
+
+   paginator = PageNumberPagination()
+   events_page = paginator.paginate_queryset(events, request)
+   serializer = EventSerializer(events_page, many=True)
+   response = paginator.get_paginated_response(serializer.data)
+  
+   return response
+
+@api_view(['GET'])
+@permission_classes([IsAuthenticated])
+def get_verified_registrations(request, pk):
+   
+   event = get_object_or_404(Event, id=pk, creator=request.user)
+  
+   verified_registrations = Registration.objects.filter(event=event, is_verified=True)
+   serializer = VerifiedRegistrationsSerializer(verified_registrations, many=True)
+  
+   return Response({
+        'seats left': event.seat_limit - verified_registrations.count(),
+        'registrations count': verified_registrations.count(),
+        'registered emails': serializer.data,
```

Comment:
read about recommended api responses
usually we use underscores in keys

Reply:
I'll read about this tonight. For now i have updated this accordingly.